h2o-docs/src/product/data-science/gam.rst

```diff
@@ -16,213 +16,136 @@ GAM supports importing and exporting `MOJOs <../save-and-load-model.html#support
+-  **knot_ids**: A string array storing frame keys/IDs that contain knot locations. Specify one value for each GAM column specified in ``gam_columns``.
 
--  `ignore_const_cols <algo-params/ignore_const_cols.html>`__: Enable this option to ignore constant
-   training columns, since no information can be gained from them. This option is defaults to true (enabled).
+-  **standardize_tp_gam_cols**: Standardize thin plate predictor columns. This option defaults to ``False``.
 
--  `score_each_iteration <algo-params/score_each_iteration.html>`__: (Optional) Enable this option to score during each iteration of the model training. This option is defaults to false (not enabled).
+-  **scale_tp_penalty_mat**: Scale penalty matrix for thin plate smoothers. This option defaults to ``False``.
 
--  `offset_column <algo-params/offset_column.html>`__: Specify a column to use as the offset; the value cannot be the same as the value for the ``weights_column``.
-   
-     **Note**: Offsets are per-row "bias values" that are used during model training. For Gaussian distributions, they can be seen as simple corrections to the response (y) column. Instead of learning to predict the response (y-row), the model learns to predict the (row) offset of the response column. For other distributions, the offset corrections are applied in the linearized space before applying the inverse link function to get the actual response values. 
+-  **bs**: An array specifying the B-Splines for each GAM predictor. You must include one value for each GAM predictor. One of:
 
--  `weights_column <algo-params/weights_column.html>`__: Specify a column to use for the observation weights, which are used for bias correction. The specified ``weights_column`` must be included in the specified ``training_frame``. *Python only*: To use a weights column when passing an H2OFrame to ``x`` instead of a list of column names, the specified ``training_frame`` must contain the specified``weights_column``. 
+    - ``0`` (default) specifies cubic regression spline. 
+    - ``1`` specifies thin plate regression with knots.
+    - ``2`` specifies monotone splines.
+    - ``3`` specifies splines of any polynomial order.
```

Comment:
please change

- ``2`` specifies monotone splines.
-  ``3`` specifies splines of any polynomial order.

to

- ``2`` specifies monotone splines (or I-splines).
-  ``3`` specifies NBVSplineTypeI M-splines (which can support any polynomial order).